### PR TITLE
fix: add total for referrals table

### PR DESCRIPTION
**Pull request by amateima**
This PR adds the total count for the referrals table data. It is needed to compute the total number of pages

### CODE REVIEW

**Review by mrice32 — APPROVED** [review]
LGTM!

File: src/modules/referral/services/queries.ts [hunk 1]
```diff
@@ -17,6 +17,15 @@ export const getReferralsQuery = () => {
   `;
 };
 
+export const getReferralsTotalQuery = () => {
+  return `
+    select count(*)
+    from deposits_mv as d
+    where d."referralAddress" = $1 or
```
Comment (nicholaspai) [inline src/modules/referral/services/queries.ts:24]: So you're saying either a referral has a referral address or a depositor address?

Comment (amateima) [reply src/modules/referral/services/queries.ts:24]: @nicholaspai 
yes. because you get 25% if the address is the depositor address

Comment (amateima) [reply src/modules/referral/services/queries.ts:24]: $1 is a particular address that's being checked

**Pull request merged**